.github/workflows/phpcs_on_pull_request.yml

```diff
@@ -3,14 +3,19 @@ name: Inspections
 jobs:
   runPHPCSInspection:
     name: Run PHPCS inspection
+    # Change below line to: `runs-on: self-hosted` if running on private repo under rtCamp org.
```

Comment:
@chandrapatel updated runner to `ubuntu-latest`, and added this comment. In case this repo is used to create a private repo, please update this action accordingly.